bin/prod-docker-build

```diff
@@ -14,5 +12,5 @@ docker build \
     --platform=linux/amd64 \
     --secret id=envs,src=.env \
     --progress=plain \
-    -t $TAG \
+    -t $DOCKER_TAG \
     .
```

Comment:
the switch to DOCKER_TAG goes along with https://github.com/safeinsights/iac/pull/34